client/src/components/Checkbox.tsx

```diff
@@ -23,18 +25,28 @@ const Checkbox = ({
   }, [checked])
 
   return (
-    <div className="checkbox">
-      <label className="d-flex align-items-center column-gap-1">
-        <input
-          className="checkbox"
-          type="checkbox"
-          ref={inputRef}
-          disabled={disabled}
-          onChange={onChange}
-        />
-        {label}
-      </label>
-    </div>
+    <Form.Check
+      className={classNames("checkbox form-check", {
+        indeterminate: checked == null
+      })}
+    >
+      <Form.Check.Input
+        className="checkbox"
+        type="checkbox"
+        ref={inputRef}
+        checked={!!checked}
```

Comment:
This will set the checkbox's `checked` always to either `true` or `false`, whereas the effect can also set it to `null`. Previously, this attribute was explicitly left out of the input. In any case, it doesn't work properly anymore now.